test/framework/integration_framework/fake_peer/network/loader_grpc.hpp

```diff
@@ -17,6 +18,20 @@ namespace integration_framework {
      public:
       explicit LoaderGrpc(const std::shared_ptr<FakePeer> &fake_peer);
 
+      bool sendBlockRequest(const std::string &dest_address,
```

Comment:
Please, add comments at least for return values of this and `sendBlocksRequest` methods. It is not immediately clear, what `bool` here and `size_t` there mean

Reply:
Please document all publically exposed methods.